security/cert_manager_operator/index.adoc

```diff
@@ -20,9 +20,16 @@ include::modules/cert-manager-request-methods.adoc[leveloffset=+1]
 //Supported versions
 include::modules/cert-manager-supported-versions.adoc[leveloffset=+1]
 
+//FIPS compliant support
+include::modules/cert-manager-fips-support.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 [id="cert-manager-operator-about_additional-resources"]
 == Additional resources
 
 * link:https://cert-manager.io/docs/[cert-manager project documentation]
+* xref:../../security/container_security/security-compliance.adoc#security-compliance[Understanding compliance]
+* xref:../../installing/installing-fips.adoc#installing-fips-mode_installing-fips[Installing a cluster in FIPS mode]
+* link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic module validation program]
+* link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance activities and government standards]
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/installing/index#installing-preparing-security[2.1.4. Do you need extra security for your cluster?]
```

Comment:
Since this is a link to OCP docs, it should probably be an xref. Omit "2.1.4." as mentioned in prev comment:
```suggestion
* xref:../../installing/installing-preparing.adoc#installing-preparing-security[Do you need extra security for your cluster?]
```